sdk/core/azure-core-test/src/main/java/com/azure/core/test/utils/TestProxyManager.java

```diff
@@ -52,41 +45,27 @@ public TestProxyManager(File recordingPath) {
      * @throws RuntimeException There was an issue starting the proxy process.
      */
     public void startProxy() {
-
         try {
-            String commandLine = "test-proxy";
-            // if we're not running in CI, construct the local path. TF_BUILD indicates Azure DevOps. CI indicates Github Actions.
-            if (runningLocally()) {
-                commandLine = Paths.get(TestProxyDownloader.getProxyDirectory().toString(),
+            // if we're not running in CI we will check to see if someone has started the proxy, and start one if not.
+            if (runningLocally() && !checkAlive(1, Duration.ofSeconds(1))) {
```

Comment:
Maybe do a few more loops? I think wasting a few more seconds for a more consistent experience would be worth it.

Reply:
I cut it down as this will just be a sleep when folks are trying to run tests, and in the very common case it's going to have to start a proxy when running locally.